src/Data/Profunctor/Linear.hs

```diff
@@ -116,3 +122,20 @@ instance Strong (,) () (->) where
 instance Strong Either Void (->) where
   first f (Left x) = Left (f x)
   first _ (Right y) = Right y
+
+-- XXX: Since CoLKleisli has uses, it might be better to replace all this
+-- with a Bif-like structure...
+--
+newtype CoLKleisli w a b = CoLKleisli { runCoLKleisli :: w a ->. b }
```

Comment:
I'd like to reiterate my dislike of mangling names like this. I'm _much_ rather have two different files and qualifed imports for the two Kleisli arrows (and Cokleisli).

Reply:
Okay sure!

Reply:
I've made two different files with qualified imports like you suggest - it's a lot cleaner.